Automattic/wp-calypso
# Inline Help: Migrate Styling

#### Changes proposed in this Pull Request

* Migrates the styling of inline help as part of #27515.

#### Testing instructions

Check out Inline Help (bottom right corner) and verify things look identical, including stuff like the checklist. 

<img width="320" alt="Screenshot 2019-04-21 at 21 42 07" src="https://user-images.githubusercontent.com/43215253/56475321-566aaf80-647e-11e9-85bb-388bfd93e9ef.png">

cc @jsnajdr, @blowery, @flootr
